src/fastoad/notebooks/02_explanation/04_OpenMDAO.ipynb

```diff
@@ -0,0 +1,1010 @@
+{
```

Comment:
<blockquote>We can see here that we have the same result as what we found in <a href="https://github.com/fast-aircraft-design/FAST-OAD/raw/91dd1f0886da2c366c84b520184b5d61072761c5/src/fastoad/notebooks/02_explanation/02_pure_python.ipynb" target="_blank">the second notebook</a>, an optimum <strong>MTOW</strong> at 999.1 kg obtained for an aspect ratio of 3.55.</blockquote><p>In the second notebook, I get 999.1kg for 3.495. And I get 999.<strong>2</strong>kg here. Do you get the same values ?</p><p>I could live with different values if the OpenMDAO-optimized value was lower. But here, I find it very confusing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Changed in dfe233f7, now OpenMDAO's results is visibly smaller

Reply:
<p>Ok, but the text need a small update, as we cannot tell results are strictly identical, especially for optimal aspect ratio.</p>

Reply:
Updated the text in 77d98d3c